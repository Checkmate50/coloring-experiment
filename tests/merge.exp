res : int;
n2 : int;
n1 : int;
x : int;
y : int;

x < n1;
y < n1;
x < n1;
n1 < n2;
n1 < res;
n2 < res;

???;
int;
???;
alloc n1;
???;
alloc res;