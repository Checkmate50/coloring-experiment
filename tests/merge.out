x;
y;
n1;
n2;
res;
